### PR TITLE
test: uri with multi path

### PR DESCRIPTION
**Pull request by avelino**
ref: #230 

## Summary by CodeRabbit

- **Tests**
	- Introduced a new test for verifying the response body when making GET requests to URIs with multiple paths.
	- Added a test `uri-with-multi-paths` to validate the response after a GET request to "/v1/hello/test/world" and related endpoints.

### CODE REVIEW

File: test/com/moclojer/resources/moclojer.yml [hunk 1]
```diff
@@ -54,3 +54,17 @@
...
+    path: /v1/hello/:username
+    response:
+      status: 200
+      headers:
+        Content-Type: application/json
+      body: >
+        {
+          "hello-v1": "{{path-params.username}}!"
+        }
+
```
[the remark this answers is not in the document]
Comment (avelino) [reply test/com/moclojer/resources/moclojer.yml:70]: written test, probably the issue you reported is ending the URI with `﻿/`, this caused confusion when registering the routes. Looking at the RESTful URI pattern, URIs should not end with `﻿/`.

File: test/com/moclojer/resources/moclojer.yml [hunk 2]
```diff
@@ -54,3 +54,29 @@
...
+# This mock register route with multi paths GET /v1/hello/:username
+- endpoint:
+    method: GET
+    path: /v1/hello/:username
+    response:
+      status: 200
+      headers:
+        Content-Type: application/json
+      body: >
```
Comment (J0sueTM) [inline test/com/moclojer/resources/moclojer.yml:66]: Just to make sure we're not letting this behind:

There're multiples way of defining a multiline string in yaml. Specifically >, |, |> and |+. Could they become a problem? :thinking:

File: test/com/moclojer/server_test.clj [hunk 3]
```diff
@@ -62,3 +62,18 @@
...
+             :body
+             (json/parse-string true)))))
+
+(deftest uri-with-multi-paths-fixed
+  (is (= {:hello-v1 "hello world!"}
+         (-> (helpers/service-fn (yaml/from-file "test/com/moclojer/resources/moclojer.yml"))
+             (response-for :get "/v1/hello")
+             :body
+             (json/parse-string true)))))
+
```
Comment (J0sueTM) [inline test/com/moclojer/server_test.clj:79]: Maybe add a test for a multi path mock with multiple variables in between?

For example a `/users/:username/data/:age`?

**Review by J0sueTM — APPROVED** [review]

File: test/com/moclojer/resources/moclojer.yml (hunk 2 above)
Comment (J0sueTM) [reply test/com/moclojer/resources/moclojer.yml:66]: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines

for reference.

File: test/com/moclojer/server_test.clj (hunk 3 above)
Comment (avelino) [reply test/com/moclojer/server_test.clj:89]: writer!

**Review by matheusfrancisco — APPROVED** [review]

**Pull request merged**